fontbe/src/avar.rs

```diff
@@ -199,4 +202,29 @@ mod tests {
             dump(to_segment_map(&axis(mappings, 0)))
         );
     }
+
```

Comment:
I suggest two more tests: an axis with min = default should not produce duplicate mappings and an axis with default = max should not produce duplicate mappings. Might have to hoist out the code to build a map vec to a fn to ease testing. IMHO this more directly validates the root cause o fthe problem than the test below, you could elect to remove it although retaining is also fine.

Reply:
> default = max

I think my current test already hits this case (max wdth == default == 100)

>  should not produce duplicate mappings

I'm confused by the "duplicate mappings"... That's not what the issue manifested itself. Do you mean, should not omit the default 0:0 avar segment map?

Reply:
IIUC the root cause of the problem is that the list of mapping tuples produced by `default_converter` can contain duplicate keys (e.g. min or max might be default). If you extract a function to produce the default mappings you can more directly expose the cause and prove it's fixed.

Reply:
ah you mean, take that out of `to_segment_map`! ok

Reply:
@rsheeter how about now https://github.com/googlefonts/fontc/pull/935/commits/07e3505ae843f135617be9e6aba225a57a59ade3 ?

Reply:
lgtm